src/packages/default/FileManager/main.css

```diff
@@ -34,3 +34,6 @@
 .ApplicationFileManagerWindow .unmounted {
   opacity : .6;
 }
+.ApplicationFileManagerWindow gui-menu-entry[data-id="SubmenuContext"] {
+  display : none;
+}
```

Comment:
This fixed the context menu being displayed, but it seemed that the Travis CI failed because of it.